test/regression_tests/shallow_water_bickley_jet_regression.jl

```diff
@@ -32,7 +31,7 @@ function run_shallow_water_regression(arch, formulation; regenerate_data = false
 
     small_amplitude = 1e-4
     
-    uⁱ(x, y, z) = ū(x, y, z) + small_amplitude * exp(-y^2) * randn()
+    uⁱ(x, y, z) = ū(x, y, z) + small_amplitude * exp(-y^2) 
```

Comment:
```suggestion
    uⁱ(x, y, z) = ū(x, y, z) + small_amplitude * exp(-y^2) * cos(8π*x/Lx)
```

why not having some `x`-dependence in initial condition? but definitely we want a deterministic one :)